sale_layout_hidden/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Sale Layout Hidden
+====================
+
+This module allows to Hide some sections in the printout of sale orders
+
+
+Usage
+=====
+
```

Comment:
"""Installing this module does not enable sale layout. You need to activate it in the sales settings. 

Then you can check the `hidden` field on sale layout categories. These categories will not be displayed. """